shell/platform/fuchsia/dart_runner/tests/startup_integration_test/hello_world/meta/hello-world.cml

```diff
@@ -0,0 +1,6 @@
+{
+    program: {
+        data: "data/hello_world",
+        runner: "dart_jit_runner"
```

Comment:
We don't specify the runner explicitly for the Flutter runner integration test so I wonder if this will create problems for you later, but I don't see any problem with specifying it explicitly. The only downside I can see is that you won't be able to run this component manually on release builds for example.

Reply:
I can change it in following PRs, it was asking for a runner to be listed when building.